frontend/views/components/Page.vue

```diff
@@ -183,13 +183,6 @@ $pagePaddingDesktop: 5.5rem;
   display: flex;
   align-items: center;
 
-  > span {
-    width: fit-content;
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-  }
-
```

Comment:
@SebinSong can you double-check this deletion?

Reply:
This style seems like to truncate the title. But based on my testing, it doesn't truncate.
Because `overflow: hidden` style, the group chat menu can't be visible. So I removed it.

Reply:
This was a recent change by @SebinSong so I think it might be important

Reply:
@taoeffect is correct. This is a bug-fix for [this](https://github.com/okTurtles/group-income/issues/1576) recently-closed issue.  These lines of code needs to be kept. @Silver-IT

Reply:
OK, I tested this and this was a requested change that @SebinSong made in a recently approved and merged PR. It's supposed to look like this:

<img width="410" alt="Screen Shot 2023-06-07 at 6 38 58 PM" src="https://github.com/okTurtles/group-income/assets/138706/587beb60-70ed-4553-94e1-f927a2ce20a8">

But now looks like this:

<img width="429" alt="Screen Shot 2023-06-07 at 6 38 40 PM" src="https://github.com/okTurtles/group-income/assets/138706/5a9031c3-956e-4ff1-b08e-fa301ed5b744">

Please undo this change, and in the future please also also double-check with the author of the code if you'd like to delete their code if the author is available and it wasn't a requested change. 🙏

Reply:
Because of this `overflow: hidden`, Issue #1627 is raised. I will try to find another way to fix both issues (#1576, #1627 ) together.

Reply:
I don't understand how #1627 is related to this, can you elaborate?

Reply:
Those styles are for the `span` element.
`Menu` is the child component of this `span` which has the style `overflow: hidden`.
So when the `Menu`'s `position` is `absolute`, the menu content can only be visible inside this span and out side of this span would be hidden.

Reply:
Check this `span` element and `[data-test="menuContent"]` element.

Reply:
Check this span element and [data-test="menuContent"] element.
![image](https://github.com/okTurtles/group-income/assets/57976479/0db5a34c-720b-4335-b36a-c188ae22baaf)

Reply:
Oh I see, understood 👍